src/chai/ManagedArray.inl

```diff
@@ -402,7 +453,7 @@ CHAI_HOST_DEVICE ManagedArray<T>::ManagedArray(T* data, CHAIDISAMBIGUATE, bool )
 #if !defined(__CUDA_ARCH__) && !defined(__HIP_DEVICE_COMPILE__)
```

Comment:
Are these macros defined outside of function bodies? Do initializer lists count as part of the function body?

Reply:
these are preprocessor macros, defined for the entire compilation unit